spec/controllers/container_group_controller_spec.rb

```diff
@@ -36,4 +36,25 @@
     expect(response.status).to eq(200)
     expect(response.body).to_not be_empty
   end
+
+  describe "#show" do
+    before do
+      EvmSpecHelper.create_guid_miq_server_zone
+      @container_group = FactoryGirl.create(:container_group)
+      @user = FactoryGirl.create(:user)
+      login_as @user
+    end
+
+    subject { get :show, :id => @container_group.id }
+
+    context "respond_with" do
+      it { is_expected.to have_http_status 200 }
+    end
+
+    context "render listnav partial" do
+      render_views
```

Comment:
@martinpovolny the point behind separated context is ```render_views```. For getting the HTTP status code, it's not necessary render the views.

How did you find that it's slow  and difficult to read ?
It's not slow, I ran both cases. To readability - this is DSL, so I see almost complete English sentences.

Only valid argument is saving 2 lines if I'll remove context block.